### PR TITLE
Help messages for new commands

### PR DESCRIPTION
**Pull request by zelima**

### CODE REVIEW

File: ckan_cloud_operator/providers/ckan/env/cli.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+from ckan_cloud_operator import logs
+from ckan_cloud_operator import kubectl
+
+from . import manager
+
+
+@click.group()
+def env():
+    '''Manage CKAN Enviroments
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/env/cli.py]:
```suggestion
    '''Manage CKAN Environments
```

File: ckan_cloud_operator/providers/ckan/instance/cli.py [hunk 2]
```diff
@@ -134,3 +134,239 @@ def delete(instance_id_or_name, no_dry_run):
...
+    Run Unix-like operating system commands into the running container
+
+    cco ckan instance shell --command='cat /etc/ckan/production.ini'
+    '''
+    pass
+
+
+@click.group()
+def sysadmin():
+    """Create or delete system administrator for CKAN instane"""
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/instance/cli.py]:
```suggestion
    """Create or delete system administrator for CKAN instance"""
```

File: ckan_cloud_operator/providers/ckan/instance/cli.py [hunk 3]
```diff
@@ -134,3 +134,239 @@ def delete(instance_id_or_name, no_dry_run):
...
+    Creates or makes given user system administrator
+
+    cco ckan instance sysadmin add USERNAME --pasword pasword --email email@email.com
+    '''
+
+@sysadmin.command('rm')
+@click.argument('USERNAME')
+def sysadmin_rm(username):
+    '''
+    Removes System administrator privilages from given user
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/instance/cli.py]:
```suggestion
    Removes System administrator privileges from given user
```

File: ckan_cloud_operator/providers/ckan/instance/cli.py [hunk 4]
```diff
@@ -134,3 +134,239 @@ def delete(instance_id_or_name, no_dry_run):
...
+def deployment_image_set(image_name, service):
+    '''
+    Force set given image for the given service
+    '''
+
+
+@click.group()
+def infra():
+    '''
+    Manage and debug CKAN related infrastucrure like SOLR Cloud and Postgres Databeses
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/instance/cli.py]:
```suggestion
    Manage and debug CKAN related infrastructure like SOLR Cloud and Postgres Databases
```

File: ckan_cloud_operator/providers/ckan/instance/cli.py [hunk 5]
```diff
@@ -134,3 +134,239 @@ def delete(instance_id_or_name, no_dry_run):
...
+    '''
+    Get database connection string
+    '''
+    pass
+
+infra.add_command(deployment_db)
+
+
+@deployment_db.command('get')
+def infra_solr_restart():
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/instance/cli.py:368]: Is this method name ok?

File: ckan_cloud_operator/providers/ckan/instance/cli.py [hunk 6]
```diff
@@ -134,3 +134,239 @@ def delete(instance_id_or_name, no_dry_run):
...
+    '''
+    get master connection string for ckan Database
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/instance/cli.py]:
```suggestion
    Get master connection string for ckan Database
```

File: docs/README.md [hunk 7]
```diff
@@ -240,7 +238,7 @@ cco ckan instance solr [COMMANDS] [[--flags]]
 
 ##### Commands
 
-- check - Check search index
+- check - Check search the index
```
Comment (pdelboca) [inline docs/README.md]:
```suggestion
- check - Check the search index
```

**Review by pdelboca — DISMISSED** [review]
@zelima lookks good! Just a few typos and a comment about one of the commands.

**Comment by zelima** [comment]
@pdelboca thanks ^_^

File: ckan_cloud_operator/providers/ckan/instance/cli.py (hunk 5 above)
Comment (zelima) [reply ckan_cloud_operator/providers/ckan/instance/cli.py:368]: Yep :) command itself is named differently

**Pull request merged**